IDE/MDK5-ARM/Projects/wolfSSL-Full/wolfsslFull.uvprojx

```diff
@@ -10,18 +10,20 @@
       <TargetName>STM32F207 Flash</TargetName>
       <ToolsetNumber>0x4</ToolsetNumber>
       <ToolsetName>ARM-ADS</ToolsetName>
+      <pCCUsed>5060528::V5.06 update 5 (build 528)::ARMCC</pCCUsed>
+      <uAC6>0</uAC6>
       <TargetOption>
         <TargetCommonOption>
-          <Device>STM32F207IGHx</Device>
+          <Device>STM32F756NGHx</Device>
```

Comment:
A lot of the changes here seem to be just switching the board the project targets. Is this necessary?